src/Trixi.jl

```diff
@@ -69,7 +69,7 @@ import SummationByPartsOperators: integrate, semidiscretize,
 
 # DGMulti solvers
 @reexport using StartUpDG: StartUpDG, Polynomial, SBP, Line, Tri, Quad, Hex, Tet
-using StartUpDG: RefElemData, MeshData, AbstractElemShape
+using StartUpDG: RefElemData, MeshData, AbstractElemShape, inverse_trace_constant
```

Comment:
```suggestion
using StartUpDG: RefElemData, MeshData, AbstractElemShape
```
You need to prepend it by `StartUpDG.` anyway to extend it. Thus, I would not bring it into scope here (since we don't use it without this prefix).